src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -2580,7 +2580,13 @@ static errno_t ipa_s2n_save_objects(struct sss_domain_info *dom,
     ret = sysdb_attrs_get_string(attrs->sysdb_attrs, SYSDB_SID_STR, &sid_str);
     if (ret != EOK) {
         DEBUG(SSSDBG_CRIT_FAILURE,
-              "Cannot find SID of object with override.\n");
+              "Cannot find SID of object.\n");
+        if (name != NULL) {
```

Comment:
In case of "name == NULL" we will  not see any message. We will get less info then before.